requirements.txt

```diff
@@ -1,13 +1,16 @@
 altair==4.2.0
-#required for toolbox but gives issues on Mac when not installed first
-chardet==4.0.0
+bokeh==2.4.1
+geopandas==0.10.2
+holoviews==1.14.6
+hvplot==0.7.3
 jupyter==1.0.0
-jupytext==1.11.1
-pre-commit==2.15.0
+jupyter-black==0.3.1
+jupytext==1.13.7
+pre-commit==2.17.0
 matplotlib==3.5.1
+numpy==1.22.3
 netCDF4==1.5.8
-#pin these packages cause some plot
-#types break down with later packages
-bokeh==2.4.1
-hvplot==0.7.3
-holoviews==1.14.6
+pandas==1.4.1
+python-dateutil==2.8.2
+rasterio==1.2.10
+xarray==2022.3.0
```

Comment:
Aren't these already included in toolbox?

Reply:
Yes, but the requirements file should contain all of the packages imported directly by the code in this repository. A very fake example, but aa-toolbox might decide to stop requiring `numpy` in which case any imports in this environment would fail